### PR TITLE
Mc 3707: Network acl

### PR DESCRIPTION
**Pull request by franzpgarcia**

### CODE REVIEW

File: cloudca/README.md [hunk 1]
```diff
@@ -87,6 +88,31 @@ The following arguments are supported:
...
+Create a network ACL.
+
+###Example usage
+```
+resource "cloudca_network_acl" "my_acl" {
+	service_code = "compute-east"
+	environment_name = "dev"
+	name = "test-acl"
+	description = "This is a test acl"
+	vpc = "8b46e2d1-bbc4-4fad-b3bd-1b25fcba4cec"
```
Comment (simongodard) [inline cloudca/README.md]: I thought we switched to vpc_id in other resources

File: cloudca/README.md [hunk 2]
```diff
@@ -87,6 +88,31 @@ The following arguments are supported:
...
+}
+```
+###Argument Reference
+The following arguments are supported:
+- service_code - (Required) Service code
+- environment_name - (Required) Name of environment
+- name - (Required) Name of the network ACL
+- description - (Required) Description of the network ACL
+- vpc - (Required) The name or ID of the vpc where the network ACL should be created
```
Comment (simongodard) [inline cloudca/README.md]: I thought we switched to vpc_id in other resources

File: cloudca/resource_cloudca.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ func GetCloudCAResourceMap() map[string]*schema.Resource {
 		"cloudca_tier":                 resourceCloudcaTier(),
 		"cloudca_port_forwarding_rule": resourceCloudcaPortForwardingRule(),
 		"cloudca_publicip":             resourceCloudcaPublicIp(),
+		"cloudca_networkacl":			resourceCloudcaNetworkAcl(),
```
Comment (simongodard) [inline cloudca/resource_cloudca.go]: weird alignment (Tabs vs Spaces?)

File: cloudca/README.md (hunk 2 above)
Comment (franzpgarcia) [inline cloudca/README.md]: Will do

File: cloudca/resource_cloudca.go (hunk 3 above)
Comment (franzpgarcia) [inline cloudca/resource_cloudca.go]: will go fmt

File: cloudca/README.md (hunk 2 above)
Comment (simongodard) [inline cloudca/README.md]: Great

**Comment by simongodard** [comment]
LGTM !

File: cloudca/resource_cloudca_network_acl.go [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+				return nil
+			}
+		}
+		return vErr
+	}
+
+	// Update the config
+	d.Set("name", acl.Name)
+	d.Set("description", acl.Description)
+	setValueOrID(d, "vpc", vpc.Name, acl.VpcId)
```
Comment (prollynomial) [inline cloudca/resource_cloudca_network_acl.go]: Is this needed anymore since we only use the VPC ID?

Comment (prollynomial) [inline cloudca/resource_cloudca_network_acl.go]: Nevermind, just saw #7

**Comment by prollynomial** [comment]
LGTM too

**Pull request merged**